### ISSUE TITLE
Use visually hidden text instead of aria-label for navigation menu button.

### ISSUE THREAD

**Issue by tochwill** [BBC-archive/psammead#2950]
**Describe the bug**
The BBC a11y check on CI is failing with the following error:

```
 * Forms: Labelling form controls: Fields must have labels or titles
    - Button has no text: //div[@id='root']/header/nav/div/div[1]/button
```

This is because the button contains no text, and relies instead on an `aria-label`. We can use the `VisuallyHiddenText` component instead, which will pass the a11y acceptance criteria.

Ensure that the correct screen reader behaviour is still present after this change is made.

**To Reproduce**
Steps to reproduce the behaviour:

**Expected behaviour**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**
